typescript/infra/config/environments/mainnet3/agent.ts

```diff
@@ -67,12 +67,12 @@ export const hyperlaneContextAgentChainConfig: AgentChainConfig<
     appchain: true,
     arbitrum: true,
     arbitrumnova: true,
+    arcadia: true,
     artela: true,
     arthera: true,
     astar: true,
     astarzkevm: true,
     aurora: true,
-    flame: true,
```

Comment:
this moved to ensure the list is in alphabetical order

i think it's out of order because the name was originally `astaria` or `astariaflame`